src/components/calcite-inline-editable/calcite-inline-editable.tsx

```diff
@@ -222,14 +223,18 @@ export class CalciteInlineEditable {
 
   private enableEditingButton: HTMLCalciteButtonElement;
 
-  private editingFocusTimeout: number;
-
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
+  transitionEnd = (event: TransitionEvent): void => {
+    if (event.propertyName === "padding-left" || event.propertyName === "padding-right") {
```

Comment:
☝️ suggestion: perhaps we should define a pattern for how we store and access transitionable properties so that we don't have to arbitrarily access them like this and could instead do something like `this.transitionableProperties.includes(event.propertyName)`.

Reply:
Agreed